data/report.txt

```diff
@@ -364,7 +364,7 @@ eo_data_complete    47
 
 
 
-EthiopiaTigrayCorrective2020 (Timesteps: 12)
+EthiopiaTigrayCorrective2020 (Timesteps: 24)
 ----------------------------------------------------------------------------
 eo_data_complete    200
 ✔ training amount: 200, positive class: 2.0%
```

Comment:
completed! I went back to delete and fix the time steps here and ran the pipeline again. ready for your eyes to look over @ivanzvonkov